server/api.ts

```diff
@@ -772,10 +775,7 @@ router.postAsync("/streamer", ensure.isStreamer, async (req, res) => {
 });
 
 /**
- * DELETE /api/streamer
- * Removes the current streamer
- * Params:
- *  - match: the _id of the match
+ * DELETE /api/streamer| "HT"
```

Comment:
?

Reply:
Oops probably a mis-paste, restored in 69f43497be6be94b4bcceb648c6491780279a936